src/Parser/FrostMage/Modules/Items/ZannesuJourney.js

```diff
@@ -0,0 +1,69 @@
+import ITEMS from 'common/ITEMS';
+import SPELLS from 'common/SPELLS';
+import Module from 'Parser/Core/Module';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import getDamageBonus from '../MageCore/GetDamageBonus';
+
+const DAMAGE_BONUS = .35;
+
+class ZannesuJourney extends Module {
+
+  static dependencies = {
+		combatants: Combatants,
+	};
+
+  damage = 0;
+  stackCount = 0;
+  previousStackCount = 0;
```

Comment:
Why not just get stackCount by querying the entity at the moment of Blizzard cast? (rather than the manual tracking happening below)

I understand that you need previousStackCount so you can see how buffed damage events are.

If you're having the problem where the buff fades before the cast event happens, you could try an event reorderer.